planemo/engine/interface.py

```diff
@@ -54,9 +54,10 @@ def cleanup(self):
 
     def run(self, runnable, job_path):
         """Run a job using a compatible artifact (workflow or tool)."""
-        self._check_can_run(runnable)
-        run_response = self._run(runnable, job_path)
-        return run_response
+        # TODO where is path comining from?
+        for runnable in for_path(path):
```

Comment:
Neither `path` nor `for_path` are defined here. Can you give me a few hints how to solve this?